sockeye/data_io.py

```diff
@@ -579,7 +579,8 @@ def get_validation_data_iter(data_loader: RawParallelDatasetLoader,
                              max_seq_len_source: int,
```

Comment:
somewhere in data_io.py we are creating the numpy / ndarrays for each bucket. These should probably also have a certain dtype (they are actually int so they should be int anyway), right now they are probably the default dtype of `mx.nd.array()`.

Reply:
Good call out, I'll take a look.